### ISSUE TITLE
HP:0031508 has overly long synonym that seems to be a description

### ISSUE THREAD

**Issue by roland-ewald** [obophenotype/human-phenotype-ontology#6205]
**HPO term**
[`HP:0031508`](https://hpo.jax.org/app/browse/term/HP:0031508)

**Bug description**
There is a single synonym defined:
> Thyroid hormones (TH) are involved in the regulation of metabolism, proliferation, and growth of most tissues. Serum TH levels are tightly controlled by the hypothalamic, pituitary, and thyroid (HPT) axis to deliver appropriate amounts of TH to target tissues. The two major THs (T3 and T4) are iodothyrosines synthesized by the thyroid gland under the control of thyrotropin/thyroid stimulating hormone (TSH), a glycoprotein heterodimer that is produced by the pituitary gland. TSH, in turn, is regulated by thyrotropin releasing hormone (TRH), a tripeptide generated by the hypothalamus that is released into its own portal system to reach the pituitary. Both the production of TRH and TSH are under negative feedback control determined by the circulating free TH concentrations.

This seems to be the _description_ of the term.

**Comment by pnrobinson** [comment]
Yep, that was supposed to be a comment -- I just fixed it, thanks!

**Comment by roland-ewald** [comment]
@pnrobinson perfect, thank you!
